packages/cli/src/cmds/init/handler.ts

```diff
@@ -23,10 +31,17 @@ export async function initHandler(args: IBeaconArgs & IGlobalArgs): Promise<Retu
 
 export async function initializeOptionsAndConfig(args: IBeaconArgs & IGlobalArgs): Promise<ReturnType> {
   const beaconPaths = getBeaconPaths(args);
+
+  const injectableBootEnrs = await getInjectableBootEnrs(beaconPaths.bootnodesFile);
```

Comment:
since this function doesn;t need to be async, may be the file can be read and merged in the similar fashion as configFile, sort of to give a unified interface

Reply:
Yup, that's true—We could merge it just like `configFile` on the `BeaconNodeOptions` constructor. This would also make the `getInjectableBootEnrs` function quite nicer.